flang/test/Examples/doloopWithFlag.f90

```diff
@@ -0,0 +1,17 @@
+! RUN: bbc --always-execute-loop-body %s -o - | tco | llc --relocation-model=pic --filetype=obj -o %temp.o
+! RUN: %CC -I%S/../.. %S/main.c -c -o %t.main.o
+! RUN: %CC %temp.o %t.main.o -L%L -lFortranRuntime -lFortranDecimal -lstdc++ -lm -o loop
+! RUN: ./loop | FileCheck %s
```

Comment:
Regarding Steve's comment, we have at least one end-to-end executable test in test/Lower.

It's hard to see this as an "example" Fortran program, since it's testing a special command line option, etc.

Can you move this file over to Lower?